src/menu.c

```diff
@@ -6595,7 +6620,7 @@ s32 is_in_two_player_adventure(void) {
  * Returns 1 if T.T. is avaliable to use, or 0 if not.
  */
 s32 is_tt_unlocked(void) {
-    return gActiveMagicCodes & 1;
+    return gActiveMagicCodes & TRUE;
```

Comment:
Same as above where TRUE likely isn't an coincidental matching number. Maybe we want to define which bits are for which active codes and create an enum or set of defines?